### DISCUSSION TITLE
Welcome to jasonelle Discussions!

### DISCUSSION THREAD

**Discussion by clsource** [jasonelle-archive/jasonelle-v2#103] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dataflownow** [comment]
Jasonette/Jasonelle seems like an incredible solution/framework.  I just wonder why there isn't more activity in this project.
The [forum](https://forum.jasonette.com/) is down.
http://web.jasonette.com/ no longer exists.
docs.jasonette.com redirects to GitHub.
[Jasonette web demo pages](https://github.com/Jasonette/Jasonette-Web) don't seem to work for me.

**Reply by clsource** [reply]
Those were services made by Ethan and sadly dissapeared with him.
This is the new forum.
most of the examples in the Jasonette web was transfered to the docs repo

https://github.com/jasonelle-archive/docs/
